docs/security/user-certs.md

```diff
@@ -85,12 +89,62 @@ of returning a valid AC for the user.
 
 To solve the problem split the bundle in the two PEM files as documented in the [referenced document](http://security.ncsa.illinois.edu/research/grid-howtos/usefulopenssl.html), so that the embedded CA chain is not used.
 
-Revoking Your User Certificate
-------------------------------
+### Revoking your user certificate
 
 If the security of your certificate or private key has been compromised, you have a responsibility to revoke the certificate. You can follow the steps [here](../security/certificate-management#osg-user-cert-revoke) to revoke your certificate. The same instructions apply if you need to revoke a certificate because your email address changed or your name has changed.
 
+
+Getting a Certificate from a Service Provider with cigetcert
+------------------------------------------------------------
+
+You may also get a user certificate from a SAML 2.0 Service Provider such as your home institution or XSEDE.
+This kind of certificate is short-lived, typically valid only for a week.
+Therefore it is more suitable for running jobs than using in your browser.
```

Comment:
Can we change "running jobs" to "accessing services at OSG sites such as compute elements or storage"?

Reply:
That's a mouthful. How about:

Reply:
Sure